examples/.ipynb_checkpoints/random_walk_tutorial-checkpoint.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
This looks like it shouldn't have been included in the commit. Remove it. You can add an `*/.ipynb_checkpoints/*` entry to the `.gitignore` file to ensure it doesn't get recommitted.